package.json

```diff
@@ -112,14 +78,14 @@
     "eslint-plugin-react": "^7.5.1",
     "final-form": "^4.2.0",
     "flow-bin": "^0.71.0",
-    "jest": "^23.6.0",
+    "jest": "^24.4.0",
     "jest-styled-components": "^6.3.1",
     "lerna": "2.11",
     "mkdirp-promise": "^5.0.1",
     "prop-types": "^15.6.0",
-    "react": "^16.2.0",
+    "react": "^16.8.4",
     "react-docgen": "^3.0.0",
-    "react-dom": "^16.2.0",
+    "react-dom": "^16.8.4",
```

Comment:
I guess this is showing up as a change because you need to rebase off master?

Reply:
I did a  `git merge --no-ff origin/master` then made another commit